### ISSUE TITLE
Do you have a schedule to support testnet?

### ISSUE THREAD

**Issue by Leon-Zhang** [blockchain/service-my-wallet-v3#395]
As title, thanks.

**Comment by jtormey** [comment]
No, there are no plans to support testnet in this project at the moment.

**Comment by Leon-Zhang** [comment]
@jtormey Thanks for answer. So how I can test my nodejs project which embedded your API? In case I don't wish to spend any real coin and a faster response?
